editor/editor_properties.cpp

```diff
@@ -328,12 +345,24 @@ EditorPropertyTextEnum::EditorPropertyTextEnum() {
 
 ///////////////////// PATH /////////////////////////
 
+void EditorPropertyPath::set_read_only(bool p_read_only) {
+	path->set_editable(!p_read_only);
+	path_edit->set_disabled(p_read_only);
+	EditorProperty::set_read_only(p_read_only);
+};
+
 void EditorPropertyPath::_path_selected(const String &p_path) {
+	if (is_read_only()) {
+		return;
+	}
```

Comment:
This can only happen when the property gets disabled while the dialog is visible, which is very unlikely.

Reply:
same above